index.bs

```diff
@@ -5008,6 +5008,12 @@ function process(numberOfFrames) {
 		if (!started) {
 			// Take note that buffer has started playing and get initial
 			// playhead position.
+			if (computedPlaybackRate &gt;= 0 &amp;&amp; loop &amp;&amp; offset &gt;= actualLoopEnd) {
```

Comment:
Maybe it would be good to replace `<pre>` with `<xmp>` here so that we don't have to escape > and & and so on.

Maybe put `loop` first to emphasize this (and below) is all about looping?

Reply:
Fixed both. Much better indeed, thanks.